index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/a12691952c.js" crossorigin="anonymous"></script>
+    <title>Portfolio David Tamayo</title>
+</head>
+<body>
+    <div class="mainpage">
+        <header class="toolbar">
+            <h2 class="welcome">Welcome</h2>
```

Comment:
- [ ] From the project requirements this part which represents your website logo should be inside a link  `<a>` tag for the convention, and semantics reasons. Please fix it :+1:

Reply:
@codecaiine when i did it an linter error showed up "button cannot appear as a descendant of a"